conode/Makefile

```diff
@@ -5,18 +5,22 @@ WEBAPP_ASSET := ../webapp/src/assets/config.toml
 
 all: docker
 
-.PHONY: conode
+.PHONY: cothority-pull conode newdb docker docker_run docker_wait docker_stop docker_start docker_clean clean
 
-conode:
-	make -C .. cothority-pull
+.DEFAULT_GOAL := docker_run
+
+utils: cothority-pull
```

Comment:
To avoid uneeded rebuild (and to use the file deps system that make provide), you can split the target:
```make
.PHONY: utils
utils: build/bcadmin build/csadmin build/phapp
build/bcadmin: $(wildcard ../conode/*.go ../conode/*/*.go)
        go build -o $@ ../conode/byzcoin/bcadmin
# usw...
```

Reply:
I prefer depending on go's vision of which files are new and whether it needs to re-compile. If I depend on these dependencies, I'm sure somebody (perhaps me) will stay with an old binary because a dependency wasn't included and then wonder why the binary is still the old one...